src/main/java/org/opensearch/performanceanalyzer/reader/ClusterManagerEventMetricsSnapshot.java

```diff
@@ -25,8 +25,9 @@
 import org.opensearch.performanceanalyzer.metrics.MetricsConfiguration;
 import org.opensearch.performanceanalyzer.metricsdb.MetricsDB;
 
-public class MasterEventMetricsSnapshot implements Removable {
-    private static final Logger LOG = LogManager.getLogger(MasterEventMetricsSnapshot.class);
+public class ClusterManagerEventMetricsSnapshot implements Removable {
+    private static final Logger LOG =
+            LogManager.getLogger(ClusterManagerEventMetricsSnapshot.class);
```

Comment:
nit : can be a single line

```suggestion
    private static final Logger LOG = LogManager.getLogger(ClusterManagerEventMetricsSnapshot.class);
```